src/AutoRest.CSharp/Mgmt/Generation/MgmtClientBaseWriter.cs

```diff
@@ -743,9 +743,20 @@ protected virtual void WriteLROMethodBody(MgmtClientOperation clientOperation, D
 
         protected virtual void WriteLROMethodBranch(MgmtRestOperation operation, IEnumerable<ParameterMapping> parameterMapping, bool async)
         {
+            var restClientName = GetRestClientName(operation);
```

Comment:
I would love for this to be consolidated into a common writer for dpg / mpg.  If we want to do that in a separate PR please open an issue to track it for future reference.

Reply:
Opened an issue here: https://github.com/Azure/autorest.csharp/issues/3106